examples/va/nfv/sriov/service-values.yaml

```diff
@@ -0,0 +1,20 @@
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  neutron:
+    customServiceConfig: |
+      [ml2]
+      mechanism_drivers = ovn,sriovnicswitch
+      [ml2_type_vlan]
+      network_vlan_ranges = sriov-phy4
+  nova:
+    scheduler:
+      customServiceConfig: |
+        [filter_scheduler]
+        enabled_filters = AvailabilityZoneFilter, ComputeFilter, ComputeCapabilitiesFilter, ImagePropertiesFilter, ServerGroupAntiAffinityFilter, ServerGroupAffinityFilter, PciPassthroughFilter, AggregateInstanceExtraSpecsFilter
```

Comment:
so this shoudl be remvoed

https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L78

we are configurting the following by default

[filter_scheduler]
available_filters = nova.scheduler.filters.all_filters
enabled_filters = ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,SameHostFilter,DifferentHostFilter,PciPassthroughFilter,NUMATopologyFilter
# until we can disabel upcalls we can't turn this off by default
# track_instance_changes = false
shuffle_best_same_weighed_hosts = true

the AvailabilityZoneFilter is deprecated and removed form bobcat.
while it is available in antelope/18 we intentionally do not enable it you are also missing the NumaToplogyFilter

while not strictly required its highly likely that sriov will be combined with CPU pinning, hugepages or VMs with a numa topology.

enabling any of those feature without the NumaToplogyFilter is incorrect.

we made sure our details in the nova operator were correct for almost all uses out of the box 

as a general rule i would avoid changing any config option in nova unless expressly need for the va such as scheduled filters.

in this case our default are more correct.